app/routes/languages.rb

```diff
@@ -70,6 +70,11 @@ def topic_not_found(topic)
         status 404
         erb :"languages/not_found", locals: { track_id: topic }
       end
+
+      def active(topic, selected_topic)
+        'active' if topic == selected_topic
+      end
```

Comment:
It was my intention that the method would still be defined in the erb file.

I didn't think I was asking to remove it from the file, just moving all the individual conditionals to a method.